From cd1ac418304faf822f819d09b498cf99cbdd1ead Mon Sep 17 00:00:00 2001
From: Luca Weiss <luca@z3ntu.xyz>
Date: Fri, 28 Sep 2018 14:46:55 +0200
Subject: [PATCH 4/4] Fix missing sentinel warning

ucurihandler.cpp: In constructor 'UCUriHandler::UCUriHandler()':
ucurihandler.cpp:78:73: error: missing sentinel in function call [-Werror=format=]
   78 |     char* path = nih_dbus_path(NULL, , applicationId.constData(), NULL);
      |                                                                         ^
---
 src/UbuntuToolkit/ucurihandler.cpp | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/UbuntuToolkit/ucurihandler.cpp b/src/UbuntuToolkit/ucurihandler.cpp
index ad752f80c..388fb4046 100644
--- a/src/UbuntuToolkit/ucurihandler.cpp
+++ b/src/UbuntuToolkit/ucurihandler.cpp
@@ -75,7 +75,7 @@ UCUriHandler::UCUriHandler()
         qWarning() << "UCUriHandler: Empty \"APP_ID\" environment variable, ignoring.";
         return;
     }
-    char* path = nih_dbus_path(NULL, "", applicationId.constData(), NULL);
+    char* path = nih_dbus_path(NULL, "", applicationId.constData(), (void*)NULL);
     objectPath = QString::fromLocal8Bit(path);
     nih_free(path);
 
-- 
2.25.2

